.gitignore

```diff
@@ -1,6 +1,8 @@
 *.pyc
 __pycache__
 
+.DS_Store
```

Comment:
.. I don't like adding OS-specific stuff like this but I guess its unavoidable (we have this in emscripten too).

Out if interest, as a mac user presumably you can this to your `~/.gitignore` and then it will apply to all the git repositories you ever user?    I've wondered in past this I/we should recommend that mac users do that.. I'm wondering if there are any downsides?

Reply:
The downside is that it is asking people to add their own .gitignores. I think it is better to have this here, so nobody will accidentally submit a PR with these files. The cost for having this here is practically zero anyways.

Reply:
Sure, I'm OK with leaving this here. 

What do you think about adding this to your `~/.gitignore` too though?   It might make it easier to work with repos that don't have this entry in their `.gitignore`?